less/type.less

```diff
@@ -238,6 +238,75 @@ ol {
   }
 }
 
+.circle-list(@color, @content) {
+  padding: 0;
+  margin: 32px 0;
+  list-style-type: none;
+  li {
+    position: relative;
+    padding-left: 40px;
+    margin-bottom: 16px;
+    strong {
+      color: @color;
+    }
+    &::before {
+      content: @content;
+      color: @color;
+      border: 1px solid @brand-smoke-plus-10;
+      font-family: "TransferWise-IconFont";
+      position: absolute;
+      left: 0;
+      width: 24px;
+      line-height: 22px;
+      margin-right: 16px;
+      font-size: 16px;
+      text-align: center;
+      font-weight: bold;
+      padding: 0 3px;
+      border-radius: 50%;
+    }
+  }
+}
+
+ul.circle-list {
+  &-checked {
+    .circle-list(@brand-green, "\e916")
+  }
+  &-сlosed {
+    .circle-list(@brand-red, "\e917")
```

Comment:
While the LESS is more concise by using the mixin, the compiler will style duplicate all the styles in the final CSS.  It would be better to have just the few styles that change (content, colour, border) in the mixin, and the other styles just once to limit file size.  If you use `.circle-list .circle-list-checked` together, you can get the general styles from the `.circle-list` class.  You can also deduplicate a lot of styling from the ol.circle-list if you take that approach.